bibtex/bib/quantum.bib

```diff
@@ -1328,6 +1415,19 @@ @Article{KW09
   Url                      = {http://dx.doi.org/10.1103/PhysRevLett.103.070504}
 }
 
+@Article{L02,
+  Title                    = {Quantum Vernam Cipher},
+  Author                   = {D. W. Leung},
```

Comment:
Hmm, I guess one could take the first names from other entries. Having full first names does make the authors more unique.